### ISSUE TITLE
No instantaneous movement?

### ISSUE THREAD

**Issue by Simplicitydown** [cjddmut/Unity-2D-Platformer-Controller#39]
Even with Time to Ground Speed set to zero, it still takes briefly holding, not tapping a key to get the player to start moving. This seems counter-intuitive to most old school platforming. I've modified the motor in lots of ways and tinkered with the source but cannot fix this. 
Otherwise the controller is everything one could dream of. 
Thank you greatly.

P.S. you guys should have a paypal donation box open

**Comment by llafuente** [comment]
Check PlayerController2D: `Input.GetAxis(PC2D.Input.HORIZONTAL)` takes some time to be 1.

Use something like

```
_motor.normalizedXMovement = Input.GetAxis(PC2D.Input.HORIZONTAL) > 0 ? 1 : (
  Input.GetAxis(PC2D.Input.HORIZONTAL) < -1 : 0
)
```

**Comment by Simplicitydown** [comment]
Unfortunately that code is not recognized. On the main page it says
"Time to Ground Speed - The time, in seconds, it will take to reach ground speed. This is used to calculate acceleration. A value of 0 mean instantaneous movement." - But this is not true.
It would seem as though there is code somewhere that is preventing instantaneous movement. No one has noticed this issue before?

**Comment by cjddmut** [comment]
If ya scroll down to the FAQs part of you'll see that this topic is covered :)

**I let go of the joystick and my GameObject isn't sliding the distance it is supposed to!**
If you're using the supplied PlayerController2D script or one of your own in which you use Input.GetAxis() then there's a built in deceleration in what Input.GetAxis() returns. This can definitely be impacting the distance the GameObject skids to a stop! To see a true skid to stop, set normalizedXMovement to zero.

Basically the controller script reads user input with Input.GetAxis() which has built in acceleration and decceleration. You can do what @llafuente suggested or change Input.GetAxis() to Input.GetRawAxis() in PlayerController2D.cs.
